unikernel_index.ml

```diff
@@ -3,10 +3,7 @@ let instance_unikernels instance_name albatross_instance_unikernels current_time
   Tyxml_html.(
     List.map
       (fun (name, unikernel) ->
-        let name =
-          Option.value ~default:"no name"
-            (Option.map Vmm_core.Name.Label.to_string (Vmm_core.Name.name name))
-        in
+        let name = Vmm_core.Name.to_string name in
```

Comment:
I don't understand this change. Doesn't it then again display ":robur-next" instead of "robur-next"?